src/app/pages/surveillance/surveillance/components/view.component.js

```diff
@@ -5,10 +5,12 @@ export const SurveillanceManagementViewComponent = {
         takeAction: '&',
     },
     controller: class SurveillanceManagementViewComponent {
-        constructor ($log, utilService) {
+        constructor ($log, authService, featureFlags, utilService) {
             'ngInject'
             this.$log = $log;
             this.certificationStatus = utilService.certificationStatus;
+            this.hasAnyRole = authService.hasAnyRole;
+            this.featureFlags = featureFlags;
```

Comment:
Most of the time we just attach the `isOn` function to the component:
```suggestion
            this.isOn= featureFlags.isOn;
```
similarly to the `hasAnyRole` function from the authService. It's not required, but it is convention